src/config/couch-peruser.rst

```diff
@@ -44,3 +44,6 @@ couch_peruser Options
 
         [couch_peruser]
         delete_dbs = false
+        
+    Note: When using jwt authorization, the provided token must include a custom _couchdb.roles=['_admin'] claim to for
```

Comment:
I'm not sure how we handle this in the rest of this documentation, so we should be consistent, but I think jwt is usually capitalized.

```suggestion
    Note: When using JWT authorization, the provided token must include a custom _couchdb.roles=['_admin'] claim to for
```